lib/src/Uncertainty/Distribution/GalambosCopula.cxx

```diff
@@ -0,0 +1,171 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The GalambosCopula distribution
+ *
+ *  Copyright 2005-2021 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include <cmath>
+#include "openturns/GalambosCopula.hxx"
+#include "openturns/RandomGenerator.hxx"
+#include "openturns/PersistentObjectFactory.hxx"
+#include "openturns/Exception.hxx"
+#include "openturns/SpecFunc.hxx"
+#include "openturns/SymbolicFunction.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+CLASSNAMEINIT(GalambosCopula)
+
+static const Factory<GalambosCopula> Factory_GalambosCopula;
+
+/* Default constructor */
+GalambosCopula::GalambosCopula()
+  : ExtremeValueCopula()
+  , theta_(0.5)
+{
+  setName("GalambosCopula");
+  // We set the dimension of the GalambosCopula distribution
+  setDimension(2);
+  // update function wrt parameters
+  updatePickandFunction();
+  // We don't know if the function is thread-safe and it may be called in parallel through computePDF()
+  setParallel(false);
```

Comment:
this should be handled in the parent class; drop the setParallel line and in ExtremeValueCopula::setPickandFunction add this line:
```
setParallel(pickandFunction.getImplementation()->isParallel());
```

Reply:
this would also simplify the JoeCopula class